OKR.md

```diff
@@ -0,0 +1,52 @@
+# Quarterly Objectives and Key Results
+
+We try to frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
+
+## 2018 Q4
+
+**go-ipfs handles large datasets (1TB++) without a sweat**
+- `P0` - It takes less than 48 hours to transfer 1TB dataset over Fast Ethernet (100Mbps)
+- `P0` @hannahhoward - List a sharded directory with 1M entries over LAN in under 1 minute, with less than a second to the first entry.
+- `PX` - There is a prototype implementation of GraphSync for go-ipfs
+- `P0` @magik6k - There is a better and more performant datastore module (e.g Badger or better)
+- `P1` - Rewrite pinning data structure to support large pinsets and flexible pin modes
+
+**The bandwidth usage is reduced significantly and is well kept under control**
```

Comment:
I like the idea of contributing to this objective, but I'm probably not the best person to own it.

Reply:
@Stebalien who do you suspect should own these OKRs? In the absence of a more qualified person (and in an effort to move important goals forward), I'm willing to take on some or all ownership. If there's a better owner, then I can work with them towards this objective.